crates/storage/db/src/implementation/mdbx/mod.rs

```diff
@@ -127,6 +157,11 @@ impl DatabaseArguments {
     pub const fn client_version(&self) -> &ClientVersion {
         &self.client_version
     }
+
+    /// Returns the geometry.
+    pub const fn geometry(&self) -> &Geometry<Range<usize>> {
```

Comment:
is this used anywhere? we access the field directly when building the environment, so I think there should be no need to have this method

Reply:
Good catch, removed.